### PR TITLE
add image and input component

### PR DESCRIPTION
**Pull request by sk-ip**
please review image and input component

### CODE REVIEW

File: component/image/image.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+/* Image */
+
+.round {
+    border-radius: 50%;
+}
+
+.responsive {
+    width: 100%;
+    max-width: 250px;
```
Comment (Aditi0505) [inline component/image/image.css]: Can convert this to rem

File: docs/image/index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        </div>
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%252F200%2522%2520class%253D%2522round%2522%2520%252F%253E"
+          style="width: 680px; height: 188px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <h2 class="cl-sub-heading">Responsive</h2>
+        <p class="cl-text">Image can be made to resize according to the viewport.</p>
+        <div class="showcase showcase-row" >
+            <img src="https://picsum.photos/200/200" class="responsive" />
```
Comment (Aditi0505) [inline docs/image/index.html]: Please add a meaningful alt description.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Everything else looks good.

File: component/input/input.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+.inp-error {
+    border: 3px solid #dc3545;
+}
+
+.inp-warning {
+    border: 3px solid #ffc107;
+}
+
+.inp-success {
+    border: 3px solid #28a745;
```
Comment (kotesh-arya) [inline component/input/input.css]: Using variables for these colours is one best practice as these colour validations are crucial for the input component.

**Review by kotesh-arya — APPROVED** [review]
Added a suggestion, rest of the code looks great 👍🏻

File: component/image/image.css (hunk 1 above)
Comment (sk-ip) [reply component/image/image.css]: ok

File: docs/image/index.html (hunk 2 above)
Comment (sk-ip) [reply docs/image/index.html]: done

File: component/input/input.css (hunk 3 above)
Comment (sk-ip) [reply component/input/input.css]: got it

**Pull request merged**